src/main/java/com/gtnewhorizons/navigator/mixins/late/xaerosworldmap/GuiMapMixin.java

```diff
@@ -130,6 +137,13 @@ protected GuiMapMixin(GuiScreen parent, GuiScreen escape) {
         }
 
         for (LayerRenderer layer : NavigatorApi.getActiveRenderersByPriority(XaeroWorldMap)) {
+            if (layer instanceof UniversalLayerRenderer universalLayerRenderer) {
+                for (XaeroRenderStep step : universalLayerRenderer.getRenderSteps()) {
```

Comment:
There should be `instanceof` check like in `MinimapRendererMixin`, shouldn't it?

Reply:
No it's not really necessary, no clue why I used it for the minimap though.